sdk/communication/Azure.Communication.Sms/README.md

```diff
@@ -47,31 +47,32 @@ SmsClient client = new SmsClient(new Uri(endpoint), tokenCredential);
 ```
 
 ## Examples
-### Send a SMS Message
+### Send a 1:1 SMS Message
 To send a SMS message, call the `Send` or `SendAsync` function from the `SmsClient`.
 ```C# Snippet:Azure_Communication_Sms_Tests_SendAsync
-SmsSendResult result = await client.SendAsync(
-   from: "+18001230000" // Phone number acquired on your Azure Communication resource
-   to: "+18005670000",
-   message: "Hi");
-Console.WriteLine($"Sms id: {result.MessageId}");
+SmsSendResult sendResult = await smsClient.SendAsync(
+    from: "<from-phone-number>",
```

Comment:
I think a comment like this would be helpful, let's them know the format
// Your E.164 formatted phone number used to send SMS

Reply:
Great Idea, I'll add that